lib/add-contributor.js

```diff
@@ -55,11 +55,11 @@ async function addContributor({
     originalSha: config.getOriginalSha(),
   };
 
-  const convention = config.get().commitConvention;
+  const { commitConvention, commitTag } = config.get();
   const prTitle = convertMessage({
-    tag: "docs",
+    tag: commitTag,
```

Comment:
@gr2m ah, now I recall why I named it `commitTag` => it's because the function signature expects an object with a `tag` property.

Reply:
sounds like we should rename the argument of the function, but we can do that in a separate PR, it's an internal refactoring. Let's go with `commitType`

Reply:
Thanks @gr2m I've fixed it now :) [feat: allow git commit type to be changed in configuration](https://github.com/all-contributors/app/pull/462/commits/0801a5a98abb360c3a50975a85679c852cc4f2d3) 

Let me know if there's anything else I should change / add before merging this PR?